Application/Platform/Gatekeeper/Authentication/Saml/SamlEVSSN.php

```diff
@@ -0,0 +1,147 @@
+                    // message.  Onelogin Toolkit supports for this endpoint the
+                    // HTTP-Redirect binding only
+                    'binding' => 'urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect',
+                ),
+                // SLO endpoint info of the IdP.
+                'singleLogoutService' => array(
+                    // URL Location of the IdP where the SP will send the SLO Request
+                    'url' => 'https://ucs-sso.connexion.evssn.de/simplesamlphp/saml2/idp/SingleLogoutService.php',
+                    // URL location of the IdP where the SP will send the SLO Response (ResponseLocation)
+                    // if not set, url for the SLO Request will be used
+                    'responseUrl' => '',
+                    // SAML protocol binding to be used when returning the <Response>
+                    // message.  Onelogin Toolkit supports for this endpoint the
+                    // HTTP-Redirect binding only
+                    'binding' => 'urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect',
+                ),
+                // Public x509 certificate of the IdP
+                'x509cert' => 'MIIFQjCCBCqgAwIBAgIBAjANBgkqhkiG9w0BAQsFADCBvTELMAkGA1UEBhMCVVMxCzAJBgNVBAgTAlVTMQswCQYDVQQHEwJVUzELMAkGA1UEChMCVVMxJDAiBgNVBAsTG1VuaXZlbnRpb24gQ29ycG9yYXRlIFNlcnZlcjE6MDgGA1UEAxMxVW5pdmVudGlvbiBDb3Jwb3JhdGUgU2VydmVyIFJvb3QgQ0EgKElEPWFEc1JjdWVkKTElMCMGCSqGSIb3DQEJARYWc3NsQGNvbm5leGlvbi5ldnNzbi5kZTAeFw0yMDAzMjAxMDM2NTZaFw0yNTAzMTkxMDM2NTZaMIGmMQswCQYDVQQGEwJVUzELMAkGA1UECBMCVVMxCzAJBgNVBAcTAlVTMQswCQYDVQQKEwJVUzEkMCIGA1UECxMbVW5pdmVudGlvbiBDb3Jwb3JhdGUgU2VydmVyMSMwIQYDVQQDExp1Y3Mtc3NvLmNvbm5leGlvbi5ldnNzbi5kZTElMCMGCSqGSIb3DQEJARYWc3NsQGNvbm5leGlvbi5ldnNzbi5kZTCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBAK9NAAogilphnZmgypebJWG54RDbDa+41cc5jnND75OtDFbMQ2Yrg3UR6x9zGqMBK7j13MhuMc9hItF8ATJwYm9eX4gWn7DshllBGKSTdgLEEPxf3zISpKKgQtbVP/k78F3nzepEpYkD/DjCi7AaBCP9TeC8Lt+v2jQl3lsstzmCsb89pQmxDwVRrf2b/hGayiuxwpcI+y5qgN1Zj33EPeWIqbKhDyPxKgDf4t44rhGKTJ9P92aaL2SIIws8JmUjo4gkBCwwJMZWyFpn2nPjb62r800VXJvWK5L40Cmyr8oY6mnOd5GejyBpgspazlRJDL97jelJQqvh7pUnPQhMXksCAwEAAaOCAWAwggFcMAkGA1UdEwQCMAAwHQYDVR0OBBYEFEtF5CQUgCXwTSfG7LPHzhZMdi3WMIHyBgNVHSMEgeowgeeAFO5cWZwCe/k1BTegP3bpTvqyLwYnoYHDpIHAMIG9MQswCQYDVQQGEwJVUzELMAkGA1UECBMCVVMxCzAJBgNVBAcTAlVTMQswCQYDVQQKEwJVUzEkMCIGA1UECxMbVW5pdmVudGlvbiBDb3Jwb3JhdGUgU2VydmVyMTowOAYDVQQDEzFVbml2ZW50aW9uIENvcnBvcmF0ZSBTZXJ2ZXIgUm9vdCBDQSAoSUQ9YURzUmN1ZWQpMSUwIwYJKoZIhvcNAQkBFhZzc2xAY29ubmV4aW9uLmV2c3NuLmRlggkA7DcM5i1JUoowCwYDVR0PBAQDAgXgMC4GA1UdEQQnMCWCGnVjcy1zc28uY29ubmV4aW9uLmV2c3NuLmRlggd1Y3Mtc3NvMA0GCSqGSIb3DQEBCwUAA4IBAQA8BzAI7Dpw9PZhLRKRGuuTRsHHSsLNZ3l0QH3dFAilmS8/VFvVLwAKNl2zcSe4s7+uwu9xAVgABXPLaYSdw2eoOL4yn8EUzStawJEi/ys1bNUQLcnDUZX7fRVT8bzqr17hfjdwbRHsR5k4F2xOlDjZCLCOeLqBze8DYonAo7s03HsXL1Jn21PN7YDyO3X0g2oGu7IcsIGP6hkv7e88p/gRSEBATjQMRHKCn8nujpXW0DvG/mKfa2+sRy7lGLpxU00Gt9/IV2BWWJxfz28M267FkfFsgfqjG7G1Z7OxHhPCOlRcPJedzup8ibOjGhOp1VS+PzU6HQbxa7Wsf4MYa6Fq',
```

Comment:
Codacy found an issue: [Line exceeds maximum limit of 180 characters; contains 1833 characters](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)